artman/tasks/publish/github.py

```diff
@@ -93,37 +96,33 @@ def execute(self, git_repo, api_name, api_version, language,
 
             # Copy the previously-generated GAPIC into the temporary
             # repository.
-            gapic_subpath = git_repo.get('gapic_subpath', '.')
-            src_path = os.path.abspath(os.path.join(gapic_code_dir, component))
-            self.exec_command(['git', 'rm', '-r', '--force',
-                               '--ignore-unmatch', gapic_subpath])
-            self.exec_command(['cp', '-rf', src_path, gapic_subpath])
-            self.exec_command(['git', 'add', gapic_subpath])
+            for path in git_repo.get('paths', ['.']):
```

Comment:
If we remove the ability to use a plain string (which I will likely do in a followup), there probably is not need for a util class.